src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -895,3 +895,18 @@ def dump_flow_result(flow_folder, prefix, flow_result=None, node_result=None):
     if output:
         with open(dump_folder / f"{prefix}.output.json", "w", encoding=DEFAULT_ENCODING) as f:
             json.dump(output, f, indent=2, ensure_ascii=False)
+
+
+def is_github_codespaces():
+    return os.environ.get("CODESPACES", None) == "true"
```

Comment:
do we have doc support for this environ name? maybe add a comment reference it?

Reply:
Yeah, please add this link as comment here
https://docs.github.com/en/codespaces/developing-in-a-codespace/default-environment-variables-for-your-codespace